packages/x-components/tests/e2e/cucumber/base-result-image.feature

```diff
@@ -1,16 +1,22 @@
 Feature: Base result image component
 
+  Background:
+    Given a results API with fallback images
+    And   a recommendations API with a known response
+    And   a next queries API
+    And   a suggestions API
+    And   a related tags API
+    And   no special config for layout view
+    And   start button is clicked
+
   Scenario Outline: 1. Placeholders are replaced for images or fallbacks in case it is not possible to load the images
-    Given no special config for base-result-image view
-    And   3 picture placeholders with no final content loaded yet
+    And   "<query>" is searched
+    Then  related results are displayed
     When  scroll down is performed
```

Comment:
Remove the scroll and use just the necessary results, it will simplify the example a lot.